### PR TITLE
Removed feature to send BrowserStacks screenshot url to TestRail and made Webapp and Mobileapp helper free from BrowserStack flag/code

### PR DESCRIPTION
**Pull request by rohandudam**
- Removed BrowserStack feature to send screenshot urls to TestRail because BrowserStack no more allowing to access session log api endpoint during the run.
- Made Web app and mobile app helpers (Base pages) free from BrowserStack flag/code

**Note:** I didn't delete complete BrowserStack Library as we need the file during BrowserStack's enhancement feature https://github.com/qxf2/qxf2-page-object-model/issues/432. In this PR I removed the outdated method related to screenshot from library.

### CODE REVIEW

File: integrations/cross_browsers/BrowserStack_Library.py
Comment (akkuldn) [inline integrations/cross_browsers/BrowserStack_Library.py:1]: Why do we have this module?

**Review by akkuldn — COMMENTED** [review]
All the tests seems to be working fine. However i noticed that for mobile tests, no matter what value we set for  `remote_browser_platform` , it still runs on Browserstack. 

- What is the purpose of `BS_ENABLE_SCREENSHOTS`? I don't notice any changes whether if it is set to true or false

File: integrations/cross_browsers/BrowserStack_Library.py
Comment (akkuldn) [reply integrations/cross_browsers/BrowserStack_Library.py:1]: Plz ignore this comment, just read the note

File: core_helpers/remote_objects.py [hunk 1]
```diff
@@ -69,14 +54,14 @@ def report_to_testrail(self,case_id,test_run_id,result_flag,msg=''):
         if self.testrail_flag is True:
             msg += '\n'.join(self.msg_list)
             msg = msg + "\n"
-            if self.browserstack_flag is True:
-                for image in self.image_url_list:
-                    msg += '\n' + '[' + image['name'] + ']('+ image['url']+')'
-                msg += '\n\n' + '[' + 'Watch Replay On BrowserStack' + ']('+ self.session_url+')'
+            # if self.browserstack_flag is True:
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply core_helpers/remote_objects.py:57]: Looks like missed it to commit file correctly. will do.

**Comment by rohandudam** [comment]
> * What is the purpose of `BS_ENABLE_SCREENSHOTS`? I don't notice any changes whether if it is set to true or false

It is updating the debug capability of BrowserStack driver. Look at https://github.com/qxf2/qxf2-page-object-model/blob/master/integrations/cross_browsers/browserstack_runner.py#L41

**Comment by rohandudam** [comment]
> * However i noticed that for mobile tests, no matter what value we set for  `remote_browser_platform` , it still runs on Browserstack.

I am able to run mobile test on SauceLab, I set REMOTE_BROWSER_PLATFORM = "SL" to run it. Can you please try it.
Your observation is true, if you set  REMOTE_BROWSER_PLATFORM value other than 'SL'. Look at the code here: https://github.com/qxf2/qxf2-page-object-model/blob/master/core_helpers/drivers/driverfactory.py#L157#L166
@akkuldn this is not a really issue and unrelated to this PR.

**Review by akkuldn — APPROVED** [review]

**Pull request merged**